deploy/backplane/srep/10-srep-readers-project.ClusterRole.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-srep-readers-project
+rules:
+# SRE can get, list common non-secret resources from everywhere
+- apiGroups:
+  - '*'
+  resources:
+  - pods
+  - services
+  - daemonsets
+  - roles
+  - deployments
+  - replicasets
+  - statefulsets
+  - jobs
+  - cronjobs
+  verbs:
+  - get
+  - list
```

Comment:
I think `watch` would also be good to include.

Reply:
:+1: